resources/js/bots/wallet/bot.js

```diff
@@ -50,9 +50,13 @@ function amountParameterBox(params, context) {
     }
 
     var txData;
-    var amount;
+    var amount = params.args[1];
```

Comment:
What happens if params.args length is 0? Also, it's possible this can be written as something like:

var amount = params.args[1] || 0

Reply:
Sure, why not. Expressions ftw!